pdgraster/WMTSCapabilitiesGenerator.py

```diff
@@ -22,37 +13,34 @@ class WMTSCapabilitiesGenerator:
         layer_title : str
         layer_identifier : str
         tile_format : str (e.g., '.png').
-        tile_matrix_set : str
+        tile_matrix_set_id : str
         tile_width : int
         tile_height : int
         max_z_level : int
         bounding_box : dict, optional
             A dictionary with keys 'left', 'right', 'bottom', and 'top' that
             specify the bounding box of the raster. If set to None, the total
             bounds of the map (global extent) are used:
-            {'left': -179.999999, 'right': 179.999999, 'bottom': -90, 'top': 90}.
+            {'left': -180, 'right': 180, 'bottom': -90, 'top': 90}.
+
 
     Usage Example:
-        config = pdgstaging.ConfigManager(config)
-        
-        generator = WMTSCapabilitiesGenerator(
-            title=config.get('title'),
-            base_url=config.get('base_url'),
-            doi=config.get('doi'),
-            layer_title=config.get('layer_title'),
-            layer_identifier=config.get('layer_identifier') ,
-            bounding_box=config.get('tile_size'),
-            tile_format=config.get('ext_web_tiles'),
-            tile_matrix_set=tms_id,
-            tile_width=config.get('tile_size')[0],
-            tile_height=config.get('tile_size')[1],
-            max_z_level=config.get('z_range')[1],
+
+    generator = WMTSCapabilitiesGenerator(
+            title="This is the title",
+            base_url="https://example.com",
+            doi="doi:10.5066/F7VQ30RM",
+            layer_title="layer_title",
+            layer_identifier="layer_identifier",
+            bounding_box=None,
```

Comment:
I like that you changed the example to hard-coded values 👍🏻 ! Makes it much more useful for people reading the docs.